.github/workflows/_test_containerized.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           # RPM-based image
           - amazonlinux:2 # pretty popular on AWS workloads
         arch: [ amd64, arm64 ]
-        go-version: [ "1.21", "1.20", "1.19" ]
+        go-version: [ "1.22-rc", "1.21", "1.20", "1.19" ]
```

Comment:
All other places in this file which refer to 1.21 probably also need a 1.22 version (`include`/`exclude`).